gcore/src/exec.rs

```diff
@@ -33,6 +33,10 @@ mod sys {
 
 /// Get the current block height.
 ///
+/// The block height serves to identify a particular block.
+/// This information can be used to enable many scenarios like restricting or
+/// allowing of some functions until this condition is reached.
```

Comment:
which this condition?

Reply:
until certain block height is reached.  - fixed